skyportal/utils/offset.py

```diff
@@ -1202,7 +1265,7 @@ def get_finding_chart(
     colors = sns.color_palette("colorblind", ncolors)
 
     start_text = [-0.45, 0.99]
-    origin = "GaiaDR2" if not used_ztfref else "ZTFref"
+    origin = "GaiaEDR3" if not used_ztfref else "ZTFref"
```

Comment:
display the correct catalog

Reply:
Maybe move this to the config? There will be GaiaDR3, 4, ...

Reply:
We'd have to upload a config file once this becomes available. So I'll not do this in config.